src/plugins/plugin.title.js

```diff
@@ -85,29 +88,23 @@ class Title extends Element {
 			me.bottom = me.height;
 		}
 
-		// Reset padding
-		me.paddingLeft = 0;
-		me.paddingTop = 0;
-		me.paddingRight = 0;
-		me.paddingBottom = 0;
-
 		// Reset minSize
 		me.minSize = {
 			width: 0,
 			height: 0
 		};
 	}
-	afterSetDimensions() {}
+	afterSetDimensions() { }
 
 	//
 
-	beforeBuildLabels() {}
-	buildLabels() {}
-	afterBuildLabels() {}
+	beforeBuildLabels() { }
+	buildLabels() { }
+	afterBuildLabels() { }
 
 	//
 
-	beforeFit() {}
+	beforeFit() { }
```

Comment:
Your editor added the spaces here? Linter does not complain, but it seems inconsistent.

Reply:
Yeah sorry, that's my code formatter which does that. Will revert.